src/utils.tsx

```diff
@@ -94,3 +95,22 @@ export function uuidV4 ():string {
     return value.toString(16)
   })
 }
+
+export function useNonInitialEffect (effect: EffectCallback, deps?: DependencyList): void {
+  const initialRender = useRef(true)
+
+  useEffect(() => {
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    let effectReturns: void | (() => void) = () => {}
+
+    if (initialRender.current) {
+      initialRender.current = false
+    } else {
+      effectReturns = effect()
+    }
+
+    if (effectReturns && typeof effectReturns === 'function') {
+      return effectReturns
```

Comment:
why don't you call effectReturns here ? `return effectReturns()`

Reply:
Good